dhcpv6/option_bootfileparam.go

```diff
@@ -0,0 +1,33 @@
+package dhcpv6
+
+import (
+	"fmt"
+)
+
+// OptBootFileParam implements the OptionBootfileParam option
+//
+// This module defines the OPT_BOOTFILE_PARAM structure.
+// https://www.ietf.org/rfc/rfc5970.txt (section 3.2)
+type OptBootFileParam string
```

Comment:
If I read the RFC properly, this should be []string instead ? You can have multiple strings in a single OPT_BOOTFILE_PARAM option, they're encoded as 16-bit length fields followed by utf-8 strings

Reply:
Khm, yeah, you right. OK, I'll fix it. Thank you.